tests/django/manage.py

```diff
@@ -29,4 +29,7 @@
 
     from django.core.management import execute_from_command_line
 
+    os.environ.setdefault(u'HYPOTHESIS_DJANGO_USETZ_TEST', u'TRUE')
```

Comment:
I don't really know - probably some interaction between conda, tox, windows, and me not grokking it yet.